diff --git a/OpenSoundMeter.pro b/OpenSoundMeter.pro
index be21bb0..500ec11 100644
--- a/OpenSoundMeter.pro
+++ b/OpenSoundMeter.pro
@@ -488,11 +488,6 @@ isEqual(GRAPH, "OPENGL") {
 
 # Special rules for deployment on Linux for AppImage
 unix:!macx:!ios:CONFIG(release, debug|release) {
-    QMAKE_POST_LINK += $$QMAKE_COPY $$PWD/OpenSoundMeter.desktop $$OUT_PWD/OpenSoundMeter_\\"$$APP_GIT_VERSION\\".desktop
-    QMAKE_POST_LINK += && $$QMAKE_COPY $$PWD/icons/white.png $$OUT_PWD
-    QMAKE_POST_LINK += && $$QMAKE_COPY $$PWD/linuxdeployosm.sh $$OUT_PWD
-    QMAKE_POST_LINK += && chmod u+x $$OUT_PWD/linuxdeployosm.sh && $$OUT_PWD/linuxdeployosm.sh $$APP_GIT_VERSION $$PWD $$[QT_INSTALL_BINS]
-
     QMAKE_CXXFLAGS_RELEASE += -fno-fast-math
 }
 
